### PR TITLE
Bump CI

### PR DESCRIPTION
**Pull request by terpstra**
Update CI in this repository to the newest methodology.

### CODE REVIEW

**Review by richardxia — APPROVED** [review]
LGTM!

**Comment by terpstra** [comment]
@bsousi5 @nategraff-sifive Just an FYI, we cannot merge this PR. It's restricted to repository code owners. If you approve, please merge.

**Pull request merged**
